docs/README.md

```diff
@@ -2,13 +2,14 @@
 
 ## Contents
 1. [API Specification](../src/api/v1/api.json)
-2. [API Migration](./api_migration.md)
-3. [Architecture](./architecture.md)
-4. [Configuration](./configuration.md)
-5. [Database](./database.md)
-6. [Error Handling](./error_handling.md)
-7. [Getting started](./getting_started.md)
-8. [Permissions](./permissions.md)
-9. [Stack](./stack.md)
-10. [Deployment](./deployment.md)
-11. [Survey Benchmarks](./survey_benchmarks.md)
+1. [API Migration](./api_migration.md)
```

Comment:
I know it won't show, but this numbering is making me sad...

Reply:
I know, me too, but I'm made happy by the thought I won't ever have to re-number any of them again.